### PR TITLE
introduce limiting analysis type

### LINKED ISSUE #165: limit "analysis type"
As a Ratings Provider who might be an RC, a TO, or a TOP
I want to indicate the analysis that set the rating
Whenever I provide a rating for any power system resource, including transmission facilities and interfaces,
So that various operations coordination activities, including transmission service request processing, can utilize only appropriate kinds of ratings/limits 
Without assuming that a given power system resources is always thermally- or stability-limited
Nor requiring that the continuous and emergency ratings be likewise constrained.

### PR DESCRIPTION
**Pull request by catkins-miso**
closes #165

### CODE REVIEW

File: docs/_data/components/schemas/limiting-analysis.yaml
Comment (catkins-miso) [inline docs/_data/components/schemas/limiting-analysis.yaml:1]: Some feedback thus far is that "constraints" in SYSTEM_SPECIFIC_CONSTRAINTS is not a good idea.

File: docs/_data/components/schemas/limiting-analysis.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  reasons.
+
+  CONTRACT_CAPACITY_LIMIT - The available facility transfer capacity is limited
+  by contractual obligations.
+
+  REGULATORY_CONSTRAINTS - The limit is imposed by regulatory requirements.
+
+  OTHER - The limit was determined by some other means not anticipated in this
+  standard.
+
```
Comment (crossMISO) [inline docs/_data/components/schemas/limiting-analysis.yaml:40]: To address a limit used in a Special Protection System (SPS) or Remedial Action Scheme (RAS), consider adding another type: "SPECIAL_PROTECTION_LIMIT" as the analysis.    The SPS or RAS limit may be a thermal (AAR or seasonal) or static limit to protect the system against a stability or cascading system issue.   

Alternatively, the "SYSTEM_SPECIFIC_CONSTRAINTS" analysis type could be used for SPS/RAS analysis.

File: docs/_data/components/schemas/limiting-analysis.yaml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  TRANSIENT_SECURITY_ASSESSMENT - Studies of the system's transient stability
+  determined the limit.
+
+  VOLTAGE_SECURITY_ASSESSMENT - Studies of the system's voltage stability
+  determined the limit.
+
+  SMALL_SIGNAL_STABILITY_ASSESSMENT - Small-signal stability studies determined
+  the limit.
+
+  SYSTEM_SPECIFIC_CONSTRAINTS - This could account for limits imposed by
```
Comment (crossMISO) [inline docs/_data/components/schemas/limiting-analysis.yaml]: add wildfire risk mitigation as an example for system_specific_constraints analysis type

**Review by crossMISO — COMMENTED** [review]
added comments about SPS/RAS (protection limits) and wildfire risk limits

File: docs/_data/components/schemas/limiting-analysis.yaml (hunk 1 above)
Comment (catkins-miso) [reply docs/_data/components/schemas/limiting-analysis.yaml:40]: I think anyone that would care about that would want to know more detail about the topology. Just saying that "a SPS action led to a different AAR proposal" may not be useful: In practice I would imagine that an operator looking at congestion would have that breaker status info available and someone looking in the archive (IMM) doesn't gain much without knowing what the topology was.
 
In sum, I think that topology is opaque to TROLIE, including any events that led to the relevant topology. What TROLIE does want to say is how the limit was determined, by sharing the constituent proposals, including information as to which security characteristics of the implicit topology led to the most limiting rating in each of those proposals.

File: docs/_data/components/schemas/limiting-analysis.yaml (hunk 2 above)
Comment (getorymckeag) [reply docs/_data/components/schemas/limiting-analysis.yaml]: Is there any value in differentiating whether the constraint is a permanent or temporary condition?  Wildfire risk is temporary for example, while I would assume sensitive loads are kind of a permanent constraint.

Comment (catkins-miso) [reply docs/_data/components/schemas/limiting-analysis.yaml]: I added ENVIRONMENTAL_CONDITIONS.

File: docs/_data/components/schemas/limiting-analysis.yaml (hunk 1 above)
Comment (catkins-miso) [reply docs/_data/components/schemas/limiting-analysis.yaml:40]: Updated the description of SYSTEM_SPECFIC to include RAS/SPS defined limits.

**Review by crossMISO — APPROVED** [review]

**Review by getorymckeag — APPROVED** [review]

**Pull request merged**